tests/integration/ps.bats

```diff
@@ -23,15 +26,12 @@ function teardown() {
 
 	runc ps test_busybox
 	[ "$status" -eq 0 ]
+	[ $EUID -ne 0 ] && lines=("${lines[@]:1}") # Remove rootless warning.
```

Comment:
Sedding `s/the actual warning message//` might be more robust

Reply:
This would be possible to do when working with `output` but in case of `lines` you can't do that, because `lines` is an indexed array, and the tests check for a specific line to contain a specific text.

Sure, we can rewrite tests to check `output` but not `lines`, but I decided to not go that route.

Overall, I think, we miss `--log-level` option, which could have been used in this case to disable warnings.